bfdd/bfdd_nb_config.c

```diff
@@ -789,7 +829,44 @@ int bfdd_bfd_sessions_single_hop_desired_echo_transmission_interval_modify(
 
 	case NB_EV_APPLY:
 		bs = nb_running_get_entry(args->dnode, NULL, true);
-		if (echo_interval == bs->timers.required_min_echo)
+		if (echo_interval == bs->timers.desired_min_echo_tx)
+			return NB_OK;
+
+		bs->peer_profile.min_echo_tx = echo_interval;
+		bfd_session_apply(bs);
+		break;
+
+	case NB_EV_ABORT:
+		/* NOTHING */
+		break;
+	}
+
+	return NB_OK;
+}
+
+/*
+ * XPath:
+ * /frr-bfdd:bfdd/bfd/sessions/single-hop/required-echo-receive-interval
+ */
+int bfdd_bfd_sessions_single_hop_required_echo_receive_interval_modify(
+	struct nb_cb_modify_args *args)
+{
+	uint32_t echo_interval = yang_dnode_get_uint32(args->dnode, NULL);
+	struct bfd_session *bs;
+
+	switch (args->event) {
+	case NB_EV_VALIDATE:
+		if (echo_interval > 60000000)
```

Comment:
Missed check here:

```c
if (echo_interval == 0)
  return NB_OK;
if (echo_interval < 10000 || echo_interval > 60000000)
  return NB_ERR_VALIDATION;
```

(like in the function above `bfdd_bfd_profile_required_echo_receive_interval_modify`)

Reply:
Thanks, fixed!